src/lips.js

```diff
@@ -988,7 +988,7 @@ defined_specials.forEach(([seq, symbol, type]) => {
    var DEBUG = false;
    function log(...args) {
    if (DEBUG) {
-   console.log(...args);
+   console.log(...args); // @jcubic can you enter what this console log returns here
```

Comment:
This is commented-out code that I used to debug Lexer it prints whatever is passed to that function as is. Like `log(10, 20, 20)` will call `console.log(10, 20, 30)`. The reason why it's a function is because of the DEBUG variable that could toggle and don't see the logs in the console.

Reply:
Should be deleted now. I made a new commit.